$MirOS: ports/devel/glib/patches/patch-glib_gutils_c,v 1.1 2005/09/23 20:17:54 bsiegert Exp $
--- glib/gutils.c.orig	2005-09-23 22:06:15.000000000 +0200
+++ glib/gutils.c	2005-09-23 22:06:22.000000000 +0200
@@ -47,7 +47,7 @@
 #include <sys/param.h>
 #endif
 #ifdef HAVE_CRT_EXTERNS_H 
-#include <crt-externs.h> /* for _NSGetEnviron */
+#include <crt_externs.h> /* for _NSGetEnviron */
 #endif
 
 /* implement gutils's inline functions
